crates/astria-sequencer/src/sequencer.rs

```diff
@@ -28,9 +28,20 @@ impl Sequencer {
     pub async fn run_until_stopped(config: Config) -> Result<()> {
         let genesis_state =
             GenesisState::from_path(config.genesis_file).context("failed reading genesis state")?;
-        let storage = penumbra_storage::TempStorage::new()
+        if config.db_filepath.try_exists()? {
```

Comment:
If you use `try_exists`, please provide error context, .e. `try_exists().context("failed checking for existence of db storage file")?`;

However, I think it's fine to just do `exists()` here because if there is some weirdness with the file then `Storage::load` will probably fail.

Reply:
The [docs](https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.exists) say `exists` is error prone and to consider `try_exists`. Is that fine in this instance?

Reply:
My thinking is that we are just doing some light reporting/guessing about what we think penumbra is going to do. I am not sure in how far the error conditions listed in the doc are relevant to us. I am wondering if they would be helpful or confusing (given that at that code point we have no insight into penumbra).